packages/edit-site/src/components/template-switcher/theme-preview.js

```diff
@@ -9,20 +9,20 @@ import { truncate } from 'lodash';
 import { __, sprintf } from '@wordpress/i18n';
 
 function ThemePreview( {
-	theme: { author_name: authorName, description, name, screenshot, version },
+	theme: { author, description, name, screenshot, version },
 } ) {
 	return (
 		<div className="edit-site-template-switcher__theme-preview">
 			<span className="edit-site-template-switcher__theme-preview-name">
-				{ name }
+				{ name.raw }
```

Comment:
Why not `rendered` here as well? We get a safe fallback if the theme doesn't have a proper name listed.

Reply:
([see](https://github.com/WordPress/gutenberg/pull/23321#discussion_r443795398))

Reply:
I think we should, that matches core's behavior see `wp_prepare_themes_for_js`.

Reply:
ec6f16d6da